app/scenes/select_team/select_team.js

```diff
@@ -55,7 +57,16 @@ export default class SelectTeam extends React.Component {
     }
 
     onSelectTeam(team) {
-        this.props.actions.handleTeamChange(team).then(this.props.actions.goBackToChannelView);
+        const {
+            goBackToChannelView,
+            closeDrawers,
+            handleTeamChange
+
+        } = this.props.actions;
+
+        handleTeamChange(team).
+        then(InteractionManager.runAfterInteractions(goBackToChannelView)).
+        then(InteractionManager.runAfterInteractions(() => setTimeout(closeDrawers, 200)));
```

Comment:
the closeDrawers wasn't being called anywhere, somehow it closes the drawer the first time, but If you switch teams again it will stay there, also this makes the transition to be smooth.

I'll change this to use async/await thou